src/cobald/daemon/config/mapping.py

```diff
@@ -109,36 +111,79 @@ def load_name(absolute_name: str):
 
 
 class SectionPlugin(Generic[M]):
-    __slots__ = "section", "digest", "required"
+    """
+    Plugin to digest a top-level configuration section
+
+    :param section: Name of the section to digest
+    :param digest: callable that receive the section
+    :param required: whether the section must be present
```

Comment:
The parameter did change, what do you think about the following:

```suggestion
    :param requirements: plugin requirements
```